scripts/export-rmib-onnx.py

```
@@ -0,0 +1,232 @@
+
+    net = Net().to(dev)
+    opt = torch.optim.AdamW(net.parameters(), lr=3e-4, weight_decay=1e-4)
+    sch = torch.optim.lr_scheduler.CosineAnnealingLR(opt, a.epochs)
+    n = S.shape[0]
+    nval = max(1, n // 10)
+    bs = 256
+    idx = torch.randperm(n)
+    S, E, Iv = S[idx], E[idx], Iv[idx]
+
+    def qb(x):                       # [...,C] → the per-joint quaternion block
+        return x.reshape(x.shape[:-1] + (J, 10))[..., 3:7]
```

Ruff (0.15.18)

[warning] Consider `(*x.shape[:-1], J, 10)` instead of concatenation

Replace with `(*x.shape[:-1], J, 10)`

(RUF005)